### ISSUE TITLE
What is different between the winblows and the console version of the wired controllers?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [GAFBlizzard/chatpad-super-driver#23]
```
What steps will reproduce the problem?
1. purchased console version of wired 360 controller from pawn shop ($20) 
knowing risk of current incompatibility.
2. Installed as directed for win7(7600 build) Ultimate for 32bit with f8 
disabled driver blah, blah....
3. started chatpad_control_i386

What is the expected output? What do you see instead?

expected output was failure due to hardware console based, not winblows.  What 
I saw instead was ...


*****
Chatpad control utility version 0.1a started.
*****
Please wait...

Control utility started.
Opening chatpad filter driver.
Successfully opened chatpad filter driver.
Successfully connected to XBox 360 controller.
Opening connection to virtual keyboard device.
Successfully opened connection to virtual keyboard device.
Successfully connected to the virtual keyboard device.
Opening connection to virtual mouse device.
Successfully opened connection to virtual mouse device.
Successfully connected to the virtual mouse device.
Microsoft initialization has been finished, so we can init the chatpad now.
IOCTL failed, lastError == 31
IOCTL failed, lastError == 31
IOCTL failed, lastError == 31
Successfully sent IOCTL.  Bytes transferred:  2
Control response data received:  0x01 0x02
Successfully sent IOCTL.  Bytes transferred:  2
Control response data sent:  0x01 0x02
Successfully sent IOCTL.  Bytes transferred:  2
Control response data received:  0x01 0x02
Successfully sent IOCTL.  Bytes transferred:  3
Reading thread started for controls data.
Reading thread started for chatpad data.
Worker thread started for mouse data.

------------------------------

Funny thing is, I expected failure, but there is something going on.  when I 
press the keys on the chatpad, the back light on the keyboard DOES LIGHT UP, 
however the input is not getting through, mouse or keyboard.

What version of the product are you using? On what operating system?

chatpad_0_0_3a_win7 ON win7(7600 build) Ultimate for 32bit

Please provide any additional information below.

Basically, I wonder if there is a timing or sequence of events issue going on 
for the PC version of the controller.  Also, does anyone know the exact 
differences between the two version of the wired controllers.  Is it possible 
to flash the firmware for the console controller to the pc firmware.


I have to admit, the idea is very sexy and I would like to help in any way 
possible.  I will start digging through the source for the 4a release, but I am 
still learning to code.


```

Original issue reported on code.google.com by `ticklesn...@gmail.com` on 23 Aug 2011 at 12:53
